modules/books.js

```diff
@@ -0,0 +1,54 @@
+const collectionBooks = [];
+
+class NewBook {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Print the info into the browser
+const printf = (input) => {
+  document.getElementById('container-book').innerHTML = input.map((items, index) => `
+      <div id="cards" class"${index}">
+      <p>"${items.title}" by ${items.author} </p>
+      <button class="buttonRemove" id="${index}">Remove</button>
+      </div>
+      `).join('');
+};
+
+// ADD book
+const add = () => {
+  const newtitle = document.querySelector('#input-title').value;
+  const newauthor = document.querySelector('#input-author').value;
+
+  // If the title or the author is empty, then don't add the book
+  if (newtitle === '' || newauthor === '') {
+    return null;
+  }
+  // Create a new Book
+  const bookInfo = new NewBook(newtitle, newauthor);
+  collectionBooks.push(bookInfo);
+
+  // Print in the html file
+  printf(collectionBooks);
+  return collectionBooks;
+};
+
+// DELET book
+const delet = (input) => {
```

Comment:
Hey, if I add multiple books, and then refresh the page and in the Book list page try to delete a book, it deletes all the books. Consider fixing it.

Reply:
Hey @muneebulrehman, thank you for your feedback. Look at this, here I tried to remove some books and the code was working properly, what should I do? And I recharge the page and it was working good also.
![photo_5156778172598037018_w](https://user-images.githubusercontent.com/97189760/183585532-1758808c-c1d6-47d1-940d-43382bc7d163.jpg)
![photo_5156778172598037017_w](https://user-images.githubusercontent.com/97189760/183585545-e9ce3967-cb66-405b-b911-0cce667ffd25.jpg)
![photo_5156778172598037016_w](https://user-images.githubusercontent.com/97189760/183585553-21b62196-10a0-4737-af24-3db36f2f4339.jpg)
![photo_5156778172598037015_w](https://user-images.githubusercontent.com/97189760/183585563-7475633f-30af-4759-9e31-ef40cfcb934d.jpg)

Reply:
@angeluray , you can always submit for a re-review or second opinion in case you feel the project is working perfectly fine on your end. It may be that there was some issue on the reviewer's side. So if you feel, everything is working perfectly on your end, consider asking for a second opinion or submitting for a re-review.

Reply:
@muneebulrehman Hi! My apologies, I asked for this review too late here and I was tired, so I couldn't understand what the problem really was. Thank you for your feedback it helped me a lot, I already solved the problem and I gonna ask for a review now.